pages/auth/login/index.tsx

```diff
@@ -1,98 +1,108 @@
-                                    <Checkbox id="rememberme1" checked={checked} onChange={(e) => setChecked(e.checked)} className="mr-2"></Checkbox>
-                                    <label htmlFor="rememberme1">Remember me</label>
-                                </div>
-                                <a className="font-medium no-underline ml-2 text-right cursor-pointer" style={{ color: 'var(--primary-color)' }}>
-                                    Forgot password?
-                                </a>
-                            </div>
-                            <Button label="Sign In" className="w-full p-3 text-xl" onClick={() => router.push('/')}></Button>
-                            <Divider />
 
-                            <Button label="Sign in with Google" icon={<GoogleIcon />} className="w-full text-xl" onClick={signInWithGoogle} />
-                        </div>
-                    </div>
+  return (
+    <div className={containerClassName}>
+      <div className="flex flex-column align-items-center justify-content-center">
+        <div
+          style={{
+            borderRadius: "56px",
+          }}
+        >
+          <div
+            className="w-full surface-card pb-7 px-5 sm:px-8"
+            style={{ borderRadius: "53px" }}
+          >
+            <div className="text-center mb-5">
+              <img
+                src={`${contextPath}/images/Seattle_Humane_Logo.png`}
+                alt="Image"
+                height="200"
+                className=""
+              />
+              <div className="">
+                <div
+                  className="text-teal-700 font-medium "
+                  style={{ position: "relative", zIndex: "10", bottom: "3rem" }}
+                >
+                  Pet Assistance and Welfare System
                 </div>
+              </div>
+            </div>
+            <div
+              className="flex"
+              style={{ position: "relative", zIndex: "11", bottom: "2rem" }}
+            >
+              <Button
+                label="Sign in with Google"
+                icon={<GoogleIcon />}
+                className="w-full text-sm mr-3 bg-white border-gray-900 text-gray-900"
+                onClick={() => {
+                  authService.signInWithGoogle();
+                }}
+              />
+              <Button
+                icon={
+                  <img
+                    src={`${contextPath}/images/MicrosoftIcon.svg`}
+                    alt="Image"
+                    height="30"
+                    className="mr-2"
+                  />
+                }
+                label="Sign in with Microsoft"
+                className="w-full text-sm bg-white border-gray-900 text-gray-900 ml-2 pl-3 pr-5"
+                onClick={() => {
+                  authService.signInWithAzure();
+                }}
+                style={{ whiteSpace: "nowrap" }}
+              ></Button>
```

Comment:
There is a tricky hydration bug happening with Button when you use the browser 'back' button (but NOT the google/microsoft 'back') to get back here. I don't know enough about hydration to know what to do about it yet, but put this as another problem to tackle on the next PR. Problem goes away when you comment away the Buttons.

Reply:
Hmm, might need to go over this. Is it expected that when we press back button from either microsoft or google suppose to go back to our log-in page?